osmosis/assetlist.json

```diff
@@ -1023,6 +1023,31 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/levana.png"
         }
       ]
+    },
+    {
+      "description": "MilkyWay's liquid staked TIA",
+      "denom_units": [
+        {
+          "denom": "factory/osmo1f5vfcph2dvfeqcqkhetwv75fda69z7e5c2dldm3kvgj23crkv6wqcn47a0/umilktia",
+          "exponent": 0
+        },
+        {
+          "denom": "milktia",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/osmo1f5vfcph2dvfeqcqkhetwv75fda69z7e5c2dldm3kvgj23crkv6wqcn47a0/umilktia",
+      "name": "milkTIA",
+      "display": "milkTIA",
```

Comment:
'milkTIA' is not defined under denom_units. (looks like this should be 'milktia')

Reply:
Hey @JeremyParish69 updating the denom_units to be milkTIA  to align